src/components/Cluster/NewCluster/ReleaseSelector/__tests__/ReleaseSelector.ts

```diff
@@ -235,4 +235,16 @@ describe('ReleaseSelector', () => {
       )
     ).toBeInTheDocument();
   });
+
+  it('has the caret hidden if the selector is not collapsible', () => {
+    renderWithStore(
+      ReleaseSelector,
+      { ...defaultProps, collapsible: false },
+      { ...defaultStoreState }
+    );
+
+    expect(
+      screen.getByText(/available releases/i).previousElementSibling
```

Comment:
This is a bit of a brittle / indirect way to check for no caret. 

Maybe we can give it a role and getByRole? https://testing-library.com/docs/dom-testing-library/api-queries#byrole

Not sure what role would be best, but I found 'role=presentation' possibly.


Giving the caret element a test-id and make sure we don't find it would also be acceptable

---- 

Presentation role: https://www.digitala11y.com/presentation-role/
Other roles: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques